openshift/templates/deploy.dc.yaml

```diff
@@ -40,16 +36,9 @@ parameters:
   - name: MEMORY_LIMIT
     description: Memory upper limit
     value: 2Gi
+  - name: WPS_WEB_GLOBAL_PROPS
```

Comment:
Can you rename this to `WPS_WEB_GLOBAL_CONFIG` or `WPS_WEB_GLOBAL_CONFIGMAP`

Reply:
done